pyproject.toml

```diff
@@ -1,7 +1,59 @@
 [build-system]
-requires = ["setuptools", "wheel", "cython", "oldest-supported-numpy"]
+requires = [
+    "cython",
+    "oldest-supported-numpy",
+    "setuptools>=61.0.0",
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "shapely"
+dynamic = ["version"]
+authors = [
+    {name = "Sean Gillies"},
```

Comment:
@sgillies did you want your email here? It was there for 1.8 ([setup.py](https://github.com/shapely/shapely/blob/maint-1.8/setup.py#L175)).

Reply:
No, name only is fine :pray: